<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JoystickDialog</class>
 <widget class="QDialog" name="JoystickDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Teleoperation</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog 
{ 
	background:  rgba(32, 80, 96, 100);
	/*background:  rgba(255, 0, 0, 255);*/
	color:white;
	font-size: 12px;
}

QPushButton
{
	color: white;
	background-color: #27a9e3;
	border-width: 0px;
	border-radius: 3px;
}

QLabel , QRadioButton, QCheckBox
{ 
	color: white; 
	border-radius: 3px;
}

QPushButton:hover, QComboBox:hover{ background-color: #66c011; }

</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,1,1,0,0,0">
   <property name="spacing">
    <number>20</number>
   </property>
   <item row="4" column="0" colspan="3">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <layout class="QHBoxLayout" name="btnHorizontalLayout" stretch="1,1,1,1">
     <item>
      <widget class="QPushButton" name="abortBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Termination</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="goOriginBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Go Origin</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetOriginBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Origin Positioning</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <layout class="QGridLayout" name="safeGridLayout" rowstretch="0,0">
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="1" column="0">
      <widget class="QPushButton" name="safeBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 200, 0)</string>
       </property>
       <property name="text">
        <string>Safe Stop</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QGroupBox" name="modeGb">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="linearRb">
          <property name="font">
           <font>
            <family>微軟正黑體</family>
            <pointsize>30</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Linear</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="jogRb">
          <property name="font">
           <font>
            <family>微軟正黑體</family>
            <pointsize>30</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Jog</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" rowspan="3" colspan="3">
    <layout class="QGridLayout" name="dirGridLayout">
     <item row="0" column="1">
      <widget class="QPushButton" name="forwardBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>50</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Forward</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="stopBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>50</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="rightBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>50</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Right</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="leftBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>50</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Left</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="backwardBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>50</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Backward</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
